values.yaml

```diff
@@ -40,7 +40,9 @@ pvc:
   mountPath: "/tmp"
 
 
-containerPort: 80
+container:
```

Comment:
@royki this is a breaking change, should be indicated by incrementing major sem ver

Reply:
bump chart version `1.0.0`